pkg/apis/gke.cattle.io/v1/types.go

```diff
@@ -22,6 +22,10 @@ import (
 
 // +genclient
 // +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +kubebuilder:printcolumn:name="ProjectID",type="string",JSONPath=".spec.projectID"
+// +kubebuilder:printcolumn:name="ClusterName",type="string",JSONPath=".spec.clusterName"
+// +kubebuilder:printcolumn:name="KubernetesVersion",type="string",JSONPath=".spec.kubernetesVersion"
+// +kubebuilder:printcolumn:name="Phase",type="string",JSONPath=".status.phase"
 
```

Comment:
```suggestion
// +kubebuilder:printcolumn:name="FailureMessage",type="string",JSONPath=".status.failureMessage"
```
One more, as this will be easier to see. Otherwise LGTM

Reply:
Thx, added it in https://github.com/rancher/gke-operator/pull/175/commits/63e5e77f9fb092c3fbe2a8ef756d57d6705d123f